docs/ref/databases.txt

```diff
@@ -362,9 +363,11 @@ Creating your database
 
 You can `create your database`_ using the command-line tools and this SQL::
 
-  CREATE DATABASE <dbname> CHARACTER SET utf8;
+  CREATE DATABASE <dbname> CHARACTER SET utf8mb4;
 
-This ensures all tables and columns will use UTF-8 by default.
+This ensures all tables and columns will use UTF-8 by default. The ``utf8``
+character set is also supported, but then only Unicode characters stored with
+up to three bytes can be stored in the database.
 
 .. _create your database: https://dev.mysql.com/doc/refman/en/create-database.html
```

Comment:
Suggested new wording, since MySQL 8 has deprecated utf8:

```
You can `create your database`_ using the command-line tools and this SQL::

  CREATE DATABASE <dbname> CHARACTER SET utf8mb4;

This ensures all tables and columns will use UTF-8 by default.

.. warning::

    Previously Django recommended the ``utf8`` character set. This cannot
    store all unicode characters, and is deprecated in MySQL 8. See the
    |mysql-utf8mb3-documentation|_, which ``utf8`` is an alias for. To
    migrate an existing database, see :ref:`mysql-convert-charset-collation`.

.. _create your database: https://dev.mysql.com/doc/refman/en/create-database.html

.. |mysql-utf8mb3-documentation| replace:: MySQL documentation on ``utf8mb3``
.. _mysql-utf8mb3-documentation: https://dev.mysql.com/doc/refman/8.0/en/charset-unicode-utf8mb3.html
```